devito/arch/compiler.py

```diff
@@ -163,9 +163,9 @@ def __init__(self):
     suffix : str, optional
         The JIT compiler version to be used. For example, assuming ``CC=gcc`` and
         ``suffix='4.9'``, the ``gcc-4.9`` will be used as JIT compiler.
-    cpp : bool, optional
+    cpp : bool, optional, default=False
```

Comment:
is this format canonical in numpydoc ?

We never use `, defaulat=...` on the first line

Reply:
Yeah it is

https://numpydoc.readthedocs.io/en/latest/format.html#parameters